DZ_1

barbara160520 opened the pull request.

Commit:
DZ_1

Commit:
Add React Devtools.

Commit:
DZ_2

Commit:
DZ_3

Commit:
DZ_4

Commit:
DZ_5

Commit:
DZ_6

Commit:
DZ_7

Commit:
DZ_8

Commit:
DZ_9

Cinium commented on src/components/Layout.js:
```diff
@@ -0,0 +1,102 @@
+                        <Box mr={2}>
+                            <PublicIcon fontSize={'large'}/>
+                        </Box>
+                        <Box class='bar'
```
className, а не class

Cinium commented on src/components/Profile.js:
```diff
@@ -0,0 +1,83 @@
+          <Typography  variant="h3">
+            Bhaumik Patel
+          </Typography>
+          <Box class="profile" >
```
className, а не class

Cinium commented on src/pages/NorFound.js:
```diff
@@ -0,0 +1,19 @@
+const NorFound = () => {
+    return (
+        
+        <Typography class="error_page">
```
className, а не class

Cinium commented on src/components/Chats.js:
```diff
@@ -0,0 +1,66 @@
+                            Ваша переписка 
+                        </Typography>
+                        </Grid>
+                        <Grid xs={6} item class="chatName">
```
className, а не class

Cinium commented on src/redux/reducers/chatsReducer/chatReducer.js:
```diff
@@ -0,0 +1,49 @@
+export const chatReducer = (state = initialState, action) => {
+    switch (action.type) {
+
+        case 'delete':
```
Имена экшнов должны быть уникальными. Этот совпадает с таковым в messageReducer

Cinium commented on src/redux/reducers/chatsReducer/chatReducer.js:
```diff
@@ -0,0 +1,49 @@
+                ...state,
+                chats: state.chats.filter((item) => item.id !== action.payload.id)
+            }
+        case "add":
```
Имена экшнов должны быть уникальными. Этот совпадает с таковым в messageReducer

Cinium commented on src/redux/reducers/messagesReducer/messageReducer.js:
```diff
@@ -0,0 +1,66 @@
+  
+export const messageReducer = (state = initialState, action) => {
+    switch(action.type){
+        case "delete":
```
Имена экшнов должны быть уникальными. Этот совпадает с таковым в chatReducer

Cinium commented on src/redux/reducers/messagesReducer/messageReducer.js:
```diff
@@ -0,0 +1,66 @@
+                ...state,
+                messages: state.messages.filter((item)=>item.id !== action.payload)
+            }
+        case "add":
```
Имена экшнов должны быть уникальными. Этот совпадает с таковым в chatReducer

Cinium commented on src/redux/reducers/chatsReducer/chatReducer.js:
```diff
@@ -0,0 +1,49 @@
+import foto_1 from '/home/barbymak/React_GB/react_app/src/images/1.jpg';
```
Не делайте проект зависимым от Ваших локальных файлов на компьютере, всё необходимое должно храниться в гит-репозитории проекта

Commit:
DZ_9.1

Добавила страницы для добавления пользователей и вывод их данных из облачной бд (вместо сообщений).  Также работает вход и регистрация. Если вход не был выполнен со страницы Блог происходит переход на страницу входа. При входе открывается доступ к Блогу, но выход не работает...

Commit:
DZ_10
